### DISCUSSION TITLE
Add Streak feature like github

### DISCUSSION THREAD

**Discussion by abhaychaturvedii** [monkeytypegame/monkeytype#4155] [Ideas] [answered]
Streak feature becoming popular among everyone . freecodecamp , codeforce , geeksforgeeks and leetcode almost every website has now streak feature and even github also has streak feature . so I think it would be great if we can add this feature .

![image](https://user-images.githubusercontent.com/55993241/230724267-929f2447-a3df-4fec-aecb-ea56cb67b4ee.png)

**Comment by Miodec** [comment, marked as answer]
Its a already a feature. Once you've typed 2 days in a row it appears on the account page, next to your name

**Comment by abhaychaturvedii** [comment]
but it's look like this 

![image](https://user-images.githubusercontent.com/55993241/230768221-77acbf98-58d7-40e8-a0b0-2686fe81787a.png)


and it's not as cool as other's streak feature

**Comment by Singh233** [comment]
Hi, I'm willing to work on this feature. Could you approve or assign me to this contribution and I will start working on it? It's a really cool feature even keybr.com has it in their profile section and it's beneficial.
